package.json

```diff
@@ -87,8 +87,10 @@
     "parse-script-tags": "^0.1.6",
     "pretty-fast": "^0.2.3",
     "prop-types": "^15.6.0",
-    "react": "16.2.0",
-    "react-dom": "16.2.0",
+    "react": "16.4.1",
```

Comment:
We need to first try to go to react 16.4 in M-C before we do it here. Is it possible to do this work with 16.3 or 16.2?

Reply:
16.3 or 16.4 is ideal because the new ref API works really nicely with how we manage focus in the Tab component.

But if it'll be a while before being able to upgrade, I'll look into ways of using the old API to achieve the same result.

Reply:
@MikeRatcliffe is looking into upgrading now. I'll have more information soon